app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/deployCloudFormationStackConfig.html

```diff
@@ -71,6 +65,38 @@ <h4>Template Configuration</h4>
       update-pipeline="ctrl.updatePipeline"
     >
     </stage-artifact-selector-delegate>
+    <div ng-if="ctrl.isChangeSet()">
+      <hr />
+      <h4>ChangeSet Configuration</h4>
+      <stage-config-field label="ChangeSet name">
+        <label class="sm-label-right">
+          <input type="text" class="form-control input-sm" ng-model="stage.changeSetName" />
+        </label>
+      </stage-config-field>
+      <stage-config-field label="" field-colums="6">
+        <label>
+          <input type="checkbox" ng-checked="ctrl.executeChangeSet()" ng-click="ctrl.toggleExecuteChangeSet()" />
+        </label>
+        <strong>Execute CloudFormation ChangeSet</strong>
+      </stage-config-field>
+      <div ng-if="ctrl.isChangeSet()">
+        <div ng-if="ctrl.executeChangeSet()">
+          <stage-config-field
+            label="If ChangeSet contains a replacement"
+            help-key="aws.cloudformation.changeSet.options"
+          >
+            <label>
+              <select class="form-control input-sm" ng-model="stage.actionOnReplacement" required>
```

Comment:
Check out `ReactSelectInput` component for this once you make as a React component